frontend/src/metabase/dashboard/components/DashboardGrid.styled.tsx

```diff
@@ -31,7 +32,7 @@ export const DashboardCard = styled.div<DashboardCardProps>`
     bottom: 0;
     right: 0;
     border-radius: 8px;
-    box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.13);
+    border: 1px solid ${colors["border"]};
```

Comment:
I'm not sure if `colors` works with whitelabeling. We should be safe if we use `color` instead:

```ts
import { color } from "metabase/lib/colors";

border: 1px solid ${color("border")};
```